data/transactions/logic/fields.go

```diff
@@ -448,6 +448,45 @@ func (s ecDsaCurveNameSpecMap) getExtraFor(name string) (extra string) {
 	return
 }
 
+// Base64Alphabet is an enum for the `base64decode` opcode
```

Comment:
Do I really need all these changes to generate the docs? Can I be more succinct?